src/codegen/sys/lib_.rs

```diff
@@ -660,12 +660,11 @@ fn generate_fields(env: &Env, struct_name: &str, fields: &[Field]) -> (Vec<Strin
                             || union_field.name.contains("padding")
                         {
                             if let Some(ref c_type) = union_field.c_type {
-                                let name = mangle_keywords(&*union_field.name);
                                 let c_type = ffi_type(env, union_field.typ, c_type);
                                 if c_type.is_err() {
                                     commented = true;
                                 }
-                                lines.push(format!("\tpub {}: {},", name, c_type.into_string()));
+                                lines.push(format!("\tpub {}: {},", &union_field.name, c_type.into_string()));
```

Comment:
We don't need the mangling anymore?

Reply:
Now mangling is done once in library postprocessing phase, instead of doing it
in every single place where field name is used. This approach doesn't retain
the original field name, but I don't think those will be useful either way.
Especially, if they had name like "priv".